python/ray/tests/test_tempfile.py

```diff
@@ -133,18 +147,25 @@ def test_raylet_tempfiles():
 
     socket_files = set(os.listdir(node.get_sockets_dir_path()))
     assert socket_files == {"plasma_store", "raylet"}
-    ray.shutdown()
 
 
-def test_tempdir_privilege():
+def test_raylet_dump_state(shutdown_only):
+    ray.init(num_cpus=1)
+    node = ray.worker._global_node
+    time.sleep(15)  # wait raylet to dump state
```

Comment:
This is a really long amount of time to add to our tests. Maybe we should make it configurable in `constants.h`.

Reply:
In order to make it shorter in the test.